docs/plugins/tv.md

```diff
@@ -0,0 +1,140 @@
+# TV
+
+The TV plugin serves as an abstraction layer to the _live TV_ functionality of a STB. The interface gives you access to the current channel
+and program information, and allows you to change the TV channel from inside an App.
+
+With the TV plugin you can adapt your App _contextually_ to what the user is currently watching on TV.
+
+## Usage
+
+In order to use the TV plugin, import it from the Lightning SDK.
+
+```js
+import { TV } from 'wpe-lightning-sdk'
+```
+
+## Available methods
+
+### Channel
+
+Retrieves information about the TV channel that is _currently_ being watched.
+
+```js
+TV.channel().then(channel => console.log(channel))
+```
+
+The `channel`-method returns a _promise_ which returns the channel information as an object.
+
+During _development_ the `channel`-method returns a random mocked channel. Optionally you can [overwrite](#overwriting-default-values) the default values.
+
+```js
+{
+  number: 1,
+  name: 'Metro News 1',
+  description: 'New York Cable News Channel',
+  entitled: true,
+}
+```
+
+### Program
+
+Retrieves the information about the TV program that is currently being watched.
+
+```js
+TV.program().then(program => console.log(program))
+```
+
+The `program`-method returns a _promise_ which returns program information as an object.
+
+During _development_ the `program`-method returns a mocked program, linked to a random mocked channel. Optionally you can [overwrite](#overwriting-default-values) the default values.
+
+```js
+{
+  title: 'The Tonight Show Starring Jimmy Fallon',
+  description: 'Late-night talk show hosted by Jimmy Fallon on NBC',
+  startTime: 'Wed, 12 Aug 2020 23:00:00 GMT', // UTC date string
+  duration: 3600, // in seconds
+  ageRating: 10,
+}
+```
+
+### Entitled
+
+Retrieves if the user is entitled to watch the current TV channel.
+
+```js
+TV.entitled().then(entitled => console.log(entitled))
+```
+
+The `entitled`-method returns a _promise_ which returns `true` when the user is entitled and `false` when not.
+
+During _development_ the `entitled`-method returns the entitlement value of the random mocked channel. Optionally you can [overwrite](#overwriting-default-values) the default values.
+
+### ChangeChannel
+
+Changes the current channel on the TV.
+
+```js
+TV.changeChannel(number).then(newChannel => console.log(newChannel))
```

Comment:
why not make this the same as the .channel API but the only difference is we're providing a number to actually manipulate the active channel? Or would that simplify things too much from a app perspective OR complicate things on the internal side?

Reply:
I've actually considered that yes and it's very simple to make that work on the internal side.
I think it looks nice, so shall we change the API to work like that (getter setter style)?